buildchain/buildchain/targets/local_image.py

```diff
@@ -84,22 +84,24 @@ def show(_task: types.Task) -> str:
 
     def _build_actions(self) -> List[types.Action]:
         """Build a container image locally."""
-        build_cmd = [
-            config.DOCKER, 'build',
-            '--tag', self.tag,
-            '--file', self.dockerfile,
-        ]
-        for arg, value in self.build_args.items():
-            build_cmd.extend(['--build-arg', '{}={}'.format(arg, value)])
-        build_cmd.append(self.dockerfile.parent)
-
-        actions : List[types.Action] = [build_cmd]
+        actions : List[types.Action] = []
+
+        # actions.append(self.check_dockerfile_dependencies)
```

Comment:
?

Dead code?

Reply:
Oops, leftover from when the branch base was the Dockerfile dependency enforcement tip.